sonar-ws/src/main/protobuf/ws-notifications.proto

```diff
@@ -35,6 +35,7 @@ message ListResponse {
 message Notification {
   optional string channel = 1;
   optional string type = 2;
+  optional string organization = 5;
```

Comment:
that's my point, functionally it makes more sense to have the organization before the component (key), ie. sort from the most generic to the most specific. Yet, organization can be number '3' for backward compatibility reasons.